### ISSUE TITLE
AttributeError running docker-compose due to removed function

### ISSUE THREAD

**Issue by nullpunktTUD** [audiolabs/webMUSHRA#59]
Trying to run ``docker-compose -f docker-compose.yml build`` I am getting the following error:

``AttributeError: module 'platform' has no attribute 'linux_distribution'``

According to some googling, this is because the function linux_distribution() was removed in Python 3.8.

Are you aware of this and/or are there any plans to support 3.8 anytime soon?

**Comment by faroit** [comment]
note, sure if this is related to our dockerfile since we don't even use python. Also I was able to build the container with newest version of docker on macOS.

If its related to the php image on linux, you might want to try `7.4-apache` instead of `7.0`...

**Comment by faroit** [comment]
@nullpunktTUD also if you are looking for a python backend: you might want to try https://github.com/nils-werner/pymushra

**Comment by nullpunktTUD** [comment]
Thanks for the quick response, but it's not about the backend. It's that
the ``docker-compose`` command from your readme failed on a brand new
Ubuntu 20.04, which ships with Python 3.8.

I tried it on Ubuntu 16.04 (using Python 3.6) and it worked fine.

After some more googling, I realize that it appears to be a docker issue.
Since I found a workaround for me, I'm not going to chase it down, though.

Long story short, I don't think it's anything you need to worry about. Feel
free to close this issue.

**Comment by faroit** [comment]
@nullpunktTUD great. If you feel we should add something to the readme, let us know (or reopen) :+1:
